components/base/ButtonLink.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <NuxtLink
```

Comment:
I think it makes sense that not everything will need to be `<NuxtLink />`. At least for now, all the places that we're using this `<ButtonLink />` is a place where it should be a `<NuxtLink />`. I think what I'd like to see instead is a new `base` component (maybe just `<Button />`) that would expose those click events, but since there is currently no use for them, I'm not sure it makes sense to add it in this PR.

Reply:
Yeah, makes sense to me. Leave that to a later PR